lib/msal-common/src/authority/CloudInstanceDiscoveryErrorResponse.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * The OpenID Configuration Endpoint Response type. Used by the authority class to get relevant OAuth endpoints.
+ */
+export type CloudInstanceDiscoveryErrorResponse = {
+    error: String;
```

Comment:
These should all be optional or write a validation function which verifies required params are present in the response.body